cli/sql_util.go

```diff
@@ -174,9 +173,18 @@ func formatVal(val interface{}) string {
 	case nil:
 		return "NULL"
 	case []byte:
-		if !utf8.Valid(t) {
-			// Ensure that protobufs containing non-UTF8 binary data print escaped.
-			return fmt.Sprintf("%q", t)
+		// Ensure that buffers containing non-printable ASCII characters print escaped.
+		printable := true
+		for _, b := range t {
+			// Printable ASCII characters are between 0x20 and 0x7E.
+			if b < 0x20 || b > 0x7E {
```

Comment:
But isn't that what you want?

Also, one of the few functional programming opportunities in Go!
```go
printable := bytes.IndexFunc(t, func(r rune) bool {
  return !unicode.IsPrint(r)
}) == -1
```